### PR TITLE
List page

### PR DESCRIPTION
**Pull request by pantageepapa**
Screenshot 2023-12-08 at 18 25 55

### CODE REVIEW

File: src/components/RestaurantCard/RestaurantCard.jsx
Comment (lydiastenflo) [inline src/components/RestaurantCard/RestaurantCard.jsx:1]: Maybe we should add the save icon + functionality here as well, so like the posts in the feed but without like and comment?

Comment (lydiastenflo) [reply src/components/RestaurantCard/RestaurantCard.jsx:1]: Love the look of the cards!! My comment is my only input but we could also merge and do that later :)

Comment (pantageepapa) [reply src/components/RestaurantCard/RestaurantCard.jsx:1]: @lydiastenflo yh I was also thinking about it but currently we only have the list view for the current user as we dont have profile pages from other friends. In this case, we do not need the functionality of saving I think.

**Review by tommasoghisini — APPROVED** [review]
looks good to me, great job!

**Pull request merged**